src/components/Editor.tsx

```diff
@@ -93,11 +93,12 @@ export const Editor = ({
     saveThemeSaveState,
     deleteThemeSaveState,
     publishThemeConfiguration,
+    sendDevThemeSaveStateData,
   } = useMessageSenders();
   useQuickFindShortcut();
 
   const buildVisualConfigLocalStorageKey = useCallback(() => {
-    if (!templateMetadata) {
+    if (!templateMetadata || templateMetadata.isThemeMode) {
```

Comment:
I don't actually think the themeMode check is necessary here or on 115, though it doesn't hurt.